src/families/bitcoin/satstack.js

```diff
@@ -16,6 +18,12 @@ import type { AccountDescriptor } from "./descriptor";
 import { getEnv } from "../../env";
 import { getCurrencyExplorer } from "../../api/Ledger";
 
+const minVersionMatch = ">=0.10.0";
```

Comment:
Can we make sure the minimum version is v0.11.1-beta, since it fixes the bug with auto detection of config file?